delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StorageScopeInitialization.scala

```diff
@@ -20,40 +22,32 @@ import ch.epfl.bluebrain.nexus.delta.sourcing.model.{EntityType, Identity, Proje
   *   the storages module
   * @param serviceAccount
   *   the subject that will be recorded when performing the initialization
-  * @param defaults
-  *   default name and description for the storage
+  * @param defaultStorageId
+  *   the id to use for the default storage to be created
+  * @param defaultStorageFields
+  *   the default value for the storage fields
   */
 class StorageScopeInitialization(
     storages: Storages,
     serviceAccount: ServiceAccount,
-    defaults: Defaults
+    defaultStorageId: Iri,
+    defaultStorageFields: StorageFields
```

Comment:
Not in the scope of this PR, but we may need to add a way for new projects to have a storage initialized in the same uniform way